pymc_marketing/mmm/preprocessing.py

```diff
@@ -5,40 +5,45 @@
 from sklearn.preprocessing import MaxAbsScaler, StandardScaler
 
 __all__ = [
-    "preprocessing_method",
+    "preprocessing_method_X",
+    "preprocessing_method_y",
     "MaxAbsScaleTarget",
     "MaxAbsScaleChannels",
     "StandardizeControls",
 ]
 
 
-def preprocessing_method(method: Callable) -> Callable:
+def preprocessing_method_X(method: Callable) -> Callable:
     if not hasattr(method, "_tags"):
         method._tags = {}  # type: ignore
-    method._tags["preprocessing"] = True  # type: ignore
+    method._tags["preprocessing_X"] = True  # type: ignore
+    return method
+
+
+def preprocessing_method_y(method: Callable) -> Callable:
+    if not hasattr(method, "_tags"):
+        method._tags = {}  # type: ignore
+    method._tags["preprocessing_y"] = True  # type: ignore
     return method
 
 
 class MaxAbsScaleTarget:
```

Comment:
Is there a path to replace these with sklearn scalers?

Reply:
I think for now we do not need to worry about the scalers and we should take this discussion to another PR :) (it is an important topic!)